articlequality/feature_lists/ptwiki.py

```diff
@@ -67,31 +67,37 @@
 
 # Links
 category_links = wikitext.revision.wikilink_titles_matching(
-    r"(Category|Categoria)\:", name="ptwiki.revision.category_links")
+    r"(Category|Categoria)\s*\:", name="ptwiki.revision.category_links")
 
 image_links = wikitext.revision.wikilink_titles_matching(
-    r"(File|Ficheiro|Arquivo|Image|Imagem)\:",
+    r"(File|Ficheiro|Arquivo|Imagem?)\s*\:",
     name="ptwiki.revision.image_links")
 
 image_templates = wikitext.revision.template_names_matching(
     r"(Scalable[ _]image|Panorama|Imagem[ _]vertical)|Panorama|Panorama 2",
     name='ptwiki.revision.image_template')
 
+side_by_side_image_templates = wikitext.revision.template_names_matching(
+    r"Imagem[ _]dupla",
+    name='ptwiki.revision.side_by_side_image_templates')
+
 
 def get_images(strs):
     """
     Parses a list of strings, expected to be tags or templates
     to get matching image patterns and returns a count of the matches.
     """
     matches = re.findall(
-        r"image[1-9]{1,2}|(File|Ficheiro|" +
-        r"Image|Imagem|Archivo):|photo[1-9][a-z]",
+        r"imagem?[1-9]{1,2}|(File|Ficheiro|" +
+        r"Imagem?|Archivo):|photo[1-9][a-z]",
         "".join(strs), re.I)
     return len(matches)
 
 
 image_templates_str = wikitext.revision.datasources.templates_str_matching(
-    r"{{(imagem multipla|galeria|multiple image|image array|gallery)",
+    r"{{(imagem[ _]multipla|galeria|multiple[ _]image|" +
+    r"image[ _]array|gallery|montagem[ _]fotográfica|" +
+    r"matriz[ _]de[ _]imagens|imagem[ _]conjunta)",
     name='ptwiki.revision.image_templates_str')
 
```

Comment:
`image[ _]array|` is superfluous since ptwiki does not have a template redirect with the name in English.

Reply:
removed.